model/src/main/kotlin/config/Excludes.kt

```diff
@@ -84,4 +84,12 @@ data class Excludes(
      * Map the project excludes by the identifiers of the provided [projects].
      */
     fun projectExcludesById(projects: Set<Project>) = projects.associate { Pair(it.id, findProjectExclude(it)) }
+
+    /**
+     * Map the scope excludes for [project] by the names of the provided [scopes].
+     */
+    fun scopeExcludesByName(project: Project, scopes: Collection<Scope>) = scopes.associate {
```

Comment:
Isn't the formatting a bit unusual here? I though in case of a non-oneliner we need to break the line after `=` and then indent the block by 8 spaces?

Reply:
I like the compactness of this formatting, but the coding conventions suggest a new line:
https://kotlinlang.org/docs/reference/coding-conventions.html#expression-body-formatting

Actually they use 4 spaces instead of 8, but that would have to be fixed separately.

Reply:
Then let's use a newline, but keep our current 8 spaces, and make that 4 spaces later? Although I believe either detekt or our (outdated) version of ktlint disagree to using 4 spaces here...